### ISSUE TITLE
There is no output of ddos -v

### ISSUE THREAD

**Issue by dikkatdeli** [jgmdev/ddos-deflate#65]
My version is CentOS Linux release 7.6.1810 (Core)

There is no output with these commands :

ddos -v -4
ddos -v4
ddos -v 4
ddos -v[4|6]
ddos -v[4]

**Comment by dazz397** [comment]
Same problem here. Did you ever figure it out?

**Comment by dikkatdeli** [comment]
> Same problem here. Did you ever figure it out?

nope. I did not.  I already removed it and used Cloudflare to protect my server.
